src/network/networkd-network.c

```diff
@@ -349,7 +349,7 @@ int network_load_one(Manager *manager, const char *filename) {
 
         network = new(Network, 1);
         if (!network)
-                return log_oom();
+                return -ENOMEM;
```

Comment:
Why drop all log_oom() rest looks good.

Reply:
Because, the function sometimes logs, and sometimes not. Anyway, I dropped the change.